modules/swagger-codegen/src/main/resources/python/model.mustache

```diff
@@ -89,9 +89,10 @@ class {{classname}}(object):
 {{/isEnum}}
 {{^isEnum}}
 {{#hasValidation}}
-
+{{#required}}
         if not {{name}}:
```

Comment:
I think it would be an issue if {{name}} is 0 (integer) or False (boolean)

What about using `if {{name}} is None` to check instead?